src/coreclr/jit/instrsxarch.h

```diff
@@ -654,6 +654,7 @@ INST3(movdqu32,          "movdqu32",          IUM_WR, SSEFLT(0x7F), BAD_CODE,
 INST3(movdqu64,          "movdqu64",          IUM_WR, SSEFLT(0x7F), BAD_CODE,     SSEFLT(0x6F),                            INS_TT_FULL_MEM,    Input_64Bit | INS_FLAGS_None)
 INST3(vinsertf64x4,      "insertf64x4",       IUM_WR, BAD_CODE,     BAD_CODE,     SSE3A(0x1A),                             INS_TT_TUPLE4,      Input_64Bit | INS_Flags_IsDstDstSrcAVXInstruction)    // Insert 256-bit packed double-precision floating point values
 INST3(vinserti64x4,      "inserti64x4",       IUM_WR, BAD_CODE,     BAD_CODE,     SSE3A(0x3A),                             INS_TT_TUPLE4,      Input_64Bit | INS_Flags_IsDstDstSrcAVXInstruction)    // Insert 256-bit packed quadword integer values
+INST3(vpternlogd,        "vpternlogd",        IUM_WR, BAD_CODE,     BAD_CODE,     SSE3A(0x25),                             INS_TT_FULL,        Input_32Bit | INS_Flags_IsDstDstSrcAVXInstruction)
```

Comment:
The way the JIT supports the VEX/EVEX encoded instructions in the disasm output is to implicitly emit a `v` prefix to the instruction when the encoding is used. So this needs to be "pternlogd" to avoid it being output as `vvpternlogd`
```suggestion
INST3(vpternlogd,        "pternlogd",         IUM_WR, BAD_CODE,     BAD_CODE,     SSE3A(0x25),                             INS_TT_FULL,        Input_32Bit | INS_Flags_IsDstDstSrcAVXInstruction)
```

Reply:
Hmmn. Seem to be printing disassembly correctly locally. Should I change and see what happens?

![image](https://user-images.githubusercontent.com/17969209/225164267-688d2cde-fa1d-478d-b265-fa76a83a9db9.png)

Reply:
There is something weird going on though. Hold on a sec. That doesn't seem right

Reply:
There is perhaps a missing check in https://github.com/dotnet/runtime/blob/main/src/coreclr/jit/instr.cpp#L104-L111 that would handle this. Most notably it's not covering `IsEvexEncodedInstruction`. The check should likely be changed to `IsVexOrEvexEncodedInstruction`

Reply:
If I understand that correctly, might have to go back and fix these too : https://github.com/dotnet/runtime/blob/main/src/coreclr/jit/instrsxarch.h#L669-L672

Reply:
Re the weird extra movups- had to fix the switch log. Didn't handle the fallthrough earlier.

![image](https://user-images.githubusercontent.com/17969209/225167642-a58030ee-1a78-4722-a935-3f89efc45bb7.png)

Reply:
Fixed the name too.